### PR TITLE
LVNETZZZ-28. Create screen for message

### PR DESCRIPTION
**Pull request by vdiakiv**
Created screen for message, added redux actions/reducer.

You can find this sub-task [here](http://ssu-jira.softserveinc.com/browse/LVNETZZZ-28)
Screenshots:
s90206-185609
s90206-185618
s90206-185711

### CODE REVIEW

**Comment by vtorkalo** [comment]
Please add screenshot of screen

File: Actions/messageActions.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+export const sendMessage = message => {
+  return (dispatch, getState) => {
+    //dispatch(validateSubject());
+    //dispatch(validateMessage());
+    const { subjectError, textError } = getState().message;
+
+    if (subjectError || textError) return;
+
+    dispatch(sendMessageBegin());
+    fetch("http://192.168.0.103:8900/api/message", {
```
Comment (vtorkalo) [inline Actions/messageActions.js]: use baseurl.js to get server address

**Review by vtorkalo — DISMISSED** [review]

File: Actions/ScheduleActions/ScheduleActionCreators.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import BASE_URL from 'TheaterSchedule/baseURL';
+import BASE_URL from '../../baseURL';
```
Comment (vtorkalo) [inline Actions/ScheduleActions/ScheduleActionCreators.js]: avoid relative path when you need go few times back

File: Screens/SettingsScreenComponents/ChooseLanguage.js [hunk 3]
```diff
@@ -10,7 +9,7 @@ import { showMessage } from "react-native-flash-message";
 
 class ChooseLanguage extends LocalizeComponent {
     state = {
-        settings: { language: "" }
+        settings: { languageCode: "" }
```
Comment (vtorkalo) [inline Screens/SettingsScreenComponents/ChooseLanguage.js]: set default language

File: Actions/ScheduleActions/ScheduleActionCreators.js (hunk 2 above)
Comment (vdiakiv) [reply Actions/ScheduleActions/ScheduleActionCreators.js]: what if I have different path?

**Review by vtorkalo — DISMISSED** [review]

**Review by vtorkalo — APPROVED** [review]

**Pull request merged**